qc_collapsed_bam/qc_collapsed_bam.cwl

```diff
@@ -0,0 +1,717 @@
+class: Workflow
+cwlVersion: v1.0
+id: qc_collapsed_bam
+label: qc_collapsed_bam
+$namespaces:
+  sbg: 'https://www.sevenbridges.com/'
+inputs:
+  - id: reference
+    type: File
+    secondaryFiles:
+      - ^.fasta.fai
+      - ^.dict
+    'sbg:x': -1379.3106689453125
+    'sbg:y': -9
+  - id: pool_b_target_intervals
+    type: File
+    label: pool_b_target_intervals
+    'sbg:x': -1380.0771484375
+    'sbg:y': -176.342529296875
+  - id: pool_a_target_intervals
+    type: File
+    label: pool_a_target_intervals
+    'sbg:x': -1407.2725830078125
+    'sbg:y': -725.6703491210938
+  - id: biometrics_vcf_file
+    type: File
+    doc: VCF file containing the SNPs to be queried.
+    'sbg:x': -1376.3106689453125
+    'sbg:y': 160.8839111328125
+  - id: collapsed_bam
+    type:
+      - File
+      - type: array
+        items: File
+    label: collapsed_bam
+    secondaryFiles:
+      - ^.bai
+    'sbg:x': -899.7366333007812
+    'sbg:y': 462.836181640625
+  - id: sample_type
+    type:
+      - 'null'
+      - string
+      - type: array
+        items: string
+    doc: 'Sample types: Normal or Tumor.'
+    'sbg:x': -900.1541137695312
+    'sbg:y': 613.905517578125
+  - id: sample_sex
+    type:
+      - 'null'
+      - string
+      - type: array
+        items: string
+    doc: Expected sample sex (i.e. M or F).
+    'sbg:x': -909.779296875
+    'sbg:y': 779.0851440429688
+  - id: sample_name
+    type:
+      - 'null'
+      - string
+      - type: array
+        items: string
+    doc: >-
+      Sample name. If not specified, sample name is automatically figured out
+      from the BAM file.
+    'sbg:x': -913.1387939453125
+    'sbg:y': 910.29150390625
+  - id: sample_group
+    type:
+      - 'null'
+      - string
+      - type: array
+        items: string
+    doc: The sample group (e.g. the sample patient ID).
+    'sbg:x': -907.56005859375
+    'sbg:y': 1060.3988037109375
+  - id: group_reads_by_umi_bam
+    type:
+      - File
+      - type: array
+        items: File
```

Comment:
Does the workflow work if this is an array of files? Or should this just be one file

Reply:
@ionox0 same thing as above. let me know if it does not make sense or if you disagree